### PR TITLE
fix(snapping): center detection

### PR DESCRIPTION
**Pull request by oguzeroglu**
The problem/solution is explained inside [this comment](https://github.com/camunda/camunda-modeler/issues/1436#issuecomment-529861783) of the issue.

Related to https://github.com/camunda/camunda-modeler/issues/1436

### CODE REVIEW

**Comment by philippfromme** [comment]
Since this PR fixes a bug a test case verifying the fix would be nice. 😉

**Comment by pinussilvestrus** [comment]
Also, it makes sense to set `master` as pull request target, since it's a bug fix.

**Comment by oguzeroglu** [comment]
@philippfromme @pinussilvestrus Done! 

I happened to recreate this bug only on the creation of a fresh diagram where the StartEvent position is (179, 99) so for the test case, in order not to interrupt the hard coded positioning logic of the existing test cases, I had to create a new diagram and use that diagram for that specific case.

**Comment by oguzeroglu** [comment]
This is the test result without the fix:
Screenshot 2019-09-11 at 11 27 28

This is the test result after the fix:
Screenshot 2019-09-11 at 11 28 53

**Comment by oguzeroglu** [comment]
@philippfromme Updated the test case!

**Comment by nikku** [comment]
Seems like center snapping, cf.

image

is now gone entirely?

capture OdxhMx_optimized

File: lib/features/snapping/BpmnConnectSnapping.js [hunk 1]
```diff
@@ -78,7 +79,7 @@ export default function BpmnConnectSnapping(eventBus, rules) {
       }
 
       if (is(target, 'bpmn:Task')) {
-        snapTargetMidOnCenter(event, target);
```
[the remark this answers is not in the document]
Comment (philippfromme) [reply lib/features/snapping/BpmnConnectSnapping.js]: You're right. We were so focused on the implementation that we missed the fact that it doesn't work for sub-processes. I'll provide an additional PR that fixes this issue.

**Comment by philippfromme** [comment]
> is now gone entirely?

What makes you think that?

**Pull request merged**